Add clearer instructions for building and testing to README (#1723)
OpenDreamProject/OpenDream: README.md

**OpenDream requires .NET 8.** You can check your version by running `dotnet --version`. It should be at least `8.0.0`.

> To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To us MSBuild, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`

Author: ```suggestion
To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To use the .NET build system, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`
```

> To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To us MSBuild, simply run `dotnet msbuild` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`

Author: ```suggestion
To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To us MSBuild, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`
```

> To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To use the .NET build system, simply run `dotnet build` in the OpenDream directory. This will build all of the solutions and put the resultant binaries in `bin`

Author: ```suggestion
To build, one can use a C# compiler (such as MSBuild) to compile the various projects described in the solution. To use the .NET build system, simply run `dotnet build -c Release` in the OpenDream directory. This will build all of the solutions in release mode and put the resultant binaries in `bin`
```
